### ISSUE TITLE
Color by Mission Type

### ISSUE THREAD

**Issue by georgiamoon** [newamericafoundation/drones#10]

**Comment by sajacy** [comment]
Can you provide the colors for each:

 `#000000` Agriculture
 `#000000` Archaeology
 `#000000` Armed Conflict Monitoring
 `#000000` Cargo
 `#000000` Community Mapping
 `#000000` Current Events
 `#000000` Disaster Response
 `#000000` Environmental Surveying
 `#000000` Search and Rescue
 `#000000` Training
 `#000000` Other
 `#000000` Wildlife Surveying

**Comment by georgiamoon** [comment]
I used colorbrewer2.org

#33a02c Agriculture
#ff7f00 Archaeology
#e31a1c Armed Conflict Monitoring
#b15928 Cargo
#a6cee3 Community Mapping
#1f78b4 Current Events
#6a3d9a Disaster Response
#cab2d6 Environmental Surveying
#fb9a99 Search and Rescue
#ffff99 Training
#fdbf6f Other
#b2df8a Wildlife Surveying

**Comment by sajacy** [comment]
addressed in #41.  The only change i made was to make the Environmental Surveying color (which was a very light purple and hard to see on the map) the old aqua color.

**Comment by kakaes** [comment]
Heya--thanks for this, and all the other stuff too! Georgia, hold off on pushing this live--think we'll need to tweak it a bit.

**Comment by sajacy** [comment]
Oh - also, if a flight had multiple "mission type" values (which is possible) it only uses the first one.

**Comment by kakaes** [comment]
Having fixed the aspect ratio the blue bar of filters overlaps (unless I zoom way out in Chrome) with the "Click to learn about a drone flight" kind of uncomfortably; going to move that to a new issue, but noticed it because the color legend also blocks out too much I think. a couple thoughts on how to fix the legend to follow here and then I'll start a new thread to deal with the overlap on top right issue... thanks jeff!

**Comment by sajacy** [comment]
No problem - I also thought the color legend might be a bit too large - maybe we can move that into the filter dropdown?  Just a thought.  Will wait on your feedback.

**Comment by kakaes** [comment]
So I think the easiest way might be to eliminate it on initial load, & replace it with a reasonably prominent button labelled "Legend" (about the same size as the "Flights/Regulations/About" tabs. Clicking on Legend woudl make somethin about the size of what we have now pop up (maybe 1 point smaller font) with a prominent-ish button to minimize it on the upper right.

Would that work?
k.

**Comment by kakaes** [comment]
Jeff-Maybe you could prioritize fixing the overlap issue (on the other thread I've started) since we can do without the legend on the live site, but having pushed the aspect ratio change to drones.newamerica.org, the live site now has the awkward overlap, which would be nice to fix sooner rather than later (though it's by no means catastrophic).
Thanks!
k.
